<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>179</width>
    <height>193</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Equality Checker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>162</width>
      <height>158</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="main_vlayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="lb_target">
       <property name="text">
        <string>Enter Expression 1 (test):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_target"/>
     </item>
     <item>
      <spacer name="vsp_inp_ctl_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lb_test">
       <property name="text">
        <string>Enter Expression 2 (target):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_test"/>
     </item>
     <item>
      <spacer name="vsp_inp_ctl">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="rb_layout">
       <item>
        <widget class="QRadioButton" name="rb_math">
         <property name="text">
          <string>Math</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rb_logic">
         <property name="text">
          <string>Logic</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="vsp_inp_ctl_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_check">
       <property name="text">
        <string>Compare</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
